### ISSUE TITLE
Some games are mislabeled or not featured despite all having pages on HLTB

### ISSUE THREAD

**Issue by Bakkestaffen** [ClassicSpeed/backloggd-hltb#7]
I'll copy all the ones I I linked to in the other thread so that you can close that one.

https://howlongtobeat.com/game/42129 and nothing https://www.backloggd.com/games/okaeri-chibi-robo-happy-richie-oosouji/

Same here: https://howlongtobeat.com/game/27879

https://www.backloggd.com/games/boku-no-natsuyasumi-2-umi-no-bouken-hen/

There are countless more games on the site which needs one. Like, there are even several Pokemon games which lacks one: https://www.backloggd.com/series/pokemon?page=1

Multiple games on this list, other than the two already mentioned: https://www.backloggd.com/u/wondershroom/list/backloggds-top-100-obscure-edition/ a few of them don't have HLTB pages, but most do.

There are even a few on the top 250 list which lacks one. Like Minecraft, likely due to it being renamed with 'Java Edition' as it's subtitle. https://www.backloggd.com/u/chump/list/backloggds-top-250?page=1

there are even several Pokemon games which lacks one: https://www.backloggd.com/series/pokemon?page=1

Several games in this franchise too: https://backloggd.com/series/mega-man-battle-network/

Multiple games on this list, other than the two already mentioned: https://www.backloggd.com/u/wondershroom/list/backloggds-top-100-obscure-edition/ a few of them don't have HLTB pages, but most do.

There are even a few on the top 250 list which lacks one. Like Minecraft, likely due to it being renamed with 'Java Edition' as it's subtitle. https://www.backloggd.com/u/chump/list/backloggds-top-250?page=1

There are also some errors. Like all Ratchet & Clank games have the same play time on them: https://backloggd.com/series/ratchet-clank/

Another franchise with this problem: https://backloggd.com/series/mario-luigi/

The Walking Dead apparently only takes 30 minutes to complete? https://backloggd.com/games/the-walking-dead/

**Comment by ClassicSpeed** [comment]
I just noticed that the search is broken/cut in half when the game name includes special characters like &. If you hover the time you can see that in the case of Mario & Luigi games is listing the times for Super Mario Bros. Wonder.

I have that working on my PC right now, and I'll upload that fix together with the ability to tell the extension what actual HLTB game it is.

In the long run I would love to be able to fix this in a better way; I know backloggd uses IGDB but I haven't found a consistent way to link entries between IGDB and HLTB.

**Comment by ClassicSpeed** [comment]
I have to do a little bit of testing and fix some stuff but this is how the feature looks right now. 


![chrome_S1noAjkaZf](https://github.com/user-attachments/assets/dc4b71ad-2157-473c-a1f0-3d8b87bc2612)

**Comment by ClassicSpeed** [comment]
I just submitted the review to both Chrome and Firefox for the new 1.2 version
You can basically click on a time badge with a question mark to add the full HLTB name and it will save it.
Also if the game does have a time, clicking will open the HLTB page on a new tab, and right-clicking will let you override what name it uses for the search.

I'll take like a day to publish but feel free to test and give any feedback on the new feature!

**Comment by Bakkestaffen** [comment]
That feature looks great, but it doesn't show up here. Do I need to update the extension? Because I see no way to do that.

**Comment by ClassicSpeed** [comment]
If you are using Chrome, you have to wait until the new version is published (You'll have to wait like a day/day and a half). It's already published on Firefox, The version is 1.2 (so you can check in your extension information to see if it's updated) It should auto-update when published.

**Comment by Bakkestaffen** [comment]
The feature has been implemented now. Can you see my contributions or are they only for me? Btw, there is one game I cant implement. Likely because it does not have a 'Main story' time. https://howlongtobeat.com/game/138159

Could you add this?

**Comment by Bakkestaffen** [comment]
Found another problem. Different games sharing same name. 'Ratchet and Clank' automatically directs to the 2016 version and not the original 2002 version. There should be a fix for this, right?

**Comment by ClassicSpeed** [comment]
> The feature has been implemented now. Can you see my contributions or are they only for me? 

The contributions are only for your own installation, at least for now. Implementing something that synchronizes game renames is not something that I would like to do.

Regarding the zero main story time and the different games with games, I found the cause and how I could potentially fix them.
I opened 2 issues regarding those two problems in order to track them. I'll try to fix them this week!

I'll close this issue since the original feature request is implemented and working.
